packages/dataviews/CHANGELOG.automattic.md

```diff
@@ -2,7 +2,7 @@
 
 ## Next
 
-- Update the field type definitions to declare the default and valid operators they support. Fields with no `type` property can use all operators and, if none is provided in the field's config, they'll display `is` and `isNot`.
+- Update the field type definitions to declare the default and valid operators they support. Fields with no `type` property can use all operators and, if none is provided in the field's config, they'll use `is` and `isNot` by default.
```

Comment:
This is just changing the changelog for the CI requirement to pass. This PR is a bugfix of this changeset, so I thought it wouldn't need a separate entry.